public/typography.css

```diff
@@ -103,3 +103,10 @@ p {
   color: var(--color-foreground);
   border-bottom-color: var(--color-foreground);
 }
+
+img.emoji {
```

Comment:
Could we change this to `.txt-emoji` and also change the class name in the twemoji options, so it aligns with the rest of the global `.txt-*` classes?

Reply:
e855171e